pyk/src/pyk/ktool/kompile.py

```diff
@@ -279,6 +279,7 @@ def __call__(
         check: bool = True,
         bug_report: BugReport | None = None,
         outer_parsed_json: bool = False,
+        ignore_warnings: Iterable[str] | None = None,
```

Comment:
Consider grouping this with the other warnings-related options, i.e.:

```python3
...
warnings_to_errors: bool = False,
ignore_warnings: Iterable[str] | None = None,
...
```

(Similarly for the argument parser, etc.)